src/main/resources/assets/traverse/lang/zh_cn.json

```diff
@@ -43,4 +54,6 @@
   "block.traverse.fir_fence_gate": "冷杉木栅栏门",
   "block.traverse.fir_door": "冷杉木门",
   "block.traverse.fir_trapdoor": "冷杉木活板门"
```

Comment:
It looks like you're missing a comma at the end of this line.

Reply:
thank